Update Xero app partnerships docs (#264)
codatio/codat-docs: docs/integrations/accounting/xero/accounting-xero-app-partner-guidance.md

---
title: "Xero App Partner guidance by use case"
description: "Understand the requirements for Financial Services, Lending, Payments, and App Store Partners"
---

> The process of becoming a Xero App Partner depends on your use case. Xero has two categories of App Partner:

Reviewer: ```suggestion
The process of becoming a Xero App Partner depends on your use case. Xero has two categories of App Partners:
```

- [Financial Services App Partners](#financial-services-partners), including [Lending App Partners](#lending-partners) and [Payments App Partners](#payments-partners)
- [Xero App Store Partners](#xero-app-store-partners), also known as _Ecosystem App Partners_

For guidance on what category applies to your use case, consult your Codat account team.

To avoid unnecessary delays, we recommend progressing the commercial and technical aspects of App Partnership in parallel, if possible.

:::caution App partnership is not always possible

> Note that Xero App Partnership is not possible for every use case.

Reviewer: ```suggestion
Note that Xero App Partnership is not possible for some use cases
```

Reviewer: or some other way to formulate this, because this can also be read as "for every use case it s not possible to get a partnership"

To become a Lending App Partner you need to:

- Implement *lending write-back* functionality within your application, as described below.
- Set up a Commercial Agreement with Xero.
- Comply with all the relevant [Xero Certification Checkpoints](https://developer.xero.com/documentation/xero-app-store/app-partner-guides/certification-checkpoints/).

### Lending write-back

Xero has one unique technical requirement for lenders: lending write-back.

The goal of the write-back is to maintain the customer's financial position in Xero at all points of the lending cycle. This is achieved by recording the loan liability, interest and/or fees, and repayments, and facilitating the reconciliation of bank accounts.

To learn about write-back in depth, see the [Xero developer documentation](https://developer.xero.com/documentation/guides/how-to-guides/general-lending-integration-guide/).

:::tip Work closely with your account team on write-back

We recommend you work closely with your Codat account team to implement lending write-back in your application. They can provide detailed guidance that is specific to the type of lending you do.

:::

### Commercial Agreement for Lending App Partners

> Xero requires a commercial agreement in order for you to become a Xero App Partner for lending. When you're ready to apply for Xero App Partnership, please contact your Codat account team and we will connect you with a relevant contact at Xero. The commercial terms of this agreement are between you as a lender and Xero. Codat is not involved in the negotiation of that commercial agreement. 

Reviewer: ```suggestion
Xero requires a commercial agreement for you to become a Xero App Partner for lending. The commercial terms of this agreement are between you as a lender and Xero. Codat is not involved in the negotiation of that commercial agreement. 
```

> When you're ready to apply for Xero App Partnership please contact your Codat account team and we'll connect you with a relevant contact at Xero.

Reviewer: Also this is a duplicate of a sentence in the paragraph above?

Author: Ah! I've removed the duplicate.

### Certification Checkpoints for Lending App Partners

Lenders are required to complete all the mandatory Certification Checkpoints required of App Partners. We've broken down the Certification Checkpoints in [Joining the Xero App Partner Program](/integrations/accounting/xero/xero-app-partner-program).

For the most efficient implementation, we suggest completing the checkpoints in the following order:

> **Lending App Partners**

Reviewer: the checkpoints - is this something Xero requires, I assume? also a bit confused why we are missing checkpoints 5 and 6 - are they not relevant for this type of partnership?

Author: Yes, the Checkpoints are required by Xero. Checkpoints 5 and 6 are not "always required" as described in "Joining the Xero App Partner Program". So, 5 & 6 are not listed here.

Author: I appreciate a numbered list of numbered checkpoints is not the optimal way to present these checkpoints. I could create an issue to improve this somehow.

Depending on the nature of your Payments solution, you may also need to implement the checkpoints required for Apps that POST or PUT data.

### FX hedging, Investments, and Financial Marketplaces

Currently, Xero does not permit App Partnership for FX hedging, Investments, and Financial Marketplaces use cases. Please consult your Codat account team for guidance if your App falls into one of those subcategories.

## Xero App Store Partners

To become an App Store Partner you need to:

- Create a Commercial Agreement with Xero.
- Comply with the relevant Certification Checkpoints.

### Commercial Agreement for App Store Partners

Xero's App Store is designed to support the go-to-market efforts of its Apps. Xero's commercial model with its App Store Partners is a referral revenue share model in which App Partners pay Xero 15% of monthly revenue (excluding fees) for net sales made through the Xero App store. 

These sales are tracked by the _Sign Up with Xero_ flow required by Xero's Certification checkpoints (Checkpoint 8). For more information about the checkpoints, see [Joining the Xero App Partner Program](/integrations/accounting/xero/xero-app-partner-program).

> When you're ready apply for Xero App Partnership please contact your Codat account team and we'll connect you with a relevant contact at Xero so you can create your commercial agreement with them.

Reviewer: ```suggestion
When you're ready to apply for Xero App Partnership, please contact your Codat account team, and we'll connect you with a relevant contact at Xero so you can create your commercial agreement with them.
```

### Certification Checkpoints for App Store Partners

> Ecosystem Partners are required to complete all the mandatory Certification Checkpoints required of App Partners. We've broken down the Certification Checkpoints in [Joining the Xero App Partner Program](/integrations/accounting/xero/xero-app-partner-program).

Reviewer: does this also need to be renamed from "ecosystem"?
